blocks/library/gutenvert/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlockType } from '../../api';
+
+registerBlockType( 'promo/gutenberg-footer', {
+	title: __( 'Editor Promo' ),
+
+	icon: 'megaphone',
+
+	category: 'common',
+
+	save() {
+		return (
+			<div>
+				<hr className="wp-block-separator" />
+				<p style={ { color: '#9e9da4', fontSize: '14px', textAlign: 'center' } }>
+					This post brought to you by the new
+					{ ' ' }<a href="https://wordpress.org/plugins/gutenberg/">Gutenberg editor</a> for
```

Comment:
Huh, is that needed?

Reply:
it's not _needed_ but I wanted to wrap the code before the `<a>` and it was awkward no matter how I diced it. could have also inserted an entity to encode the space but I felt like the literal JS space was more clear. thoughts? (without it we lose the space between `new` and the link text)